src/pyfaf/storage/problem.py

```diff
@@ -96,7 +96,7 @@ def type(self):
 
     @property
     def reports_count(self):
-        return sum(map(lambda x: x.count, self.reports))
+        return sum([x.count for x in self.reports])
```

Comment:
This is not necessary as sum() can accept iterator. On the other hand list comprehension is probably more readable.

Reply:
Mirek, thank you for the review. My understanding is that you agree with the list comprehension, so I keep the change.